test/fixtures/url-fixture/src/main/java/fixture/url/URLFixture.java

```diff
@@ -18,13 +18,15 @@
 import java.nio.file.Paths;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 /**
  * This {@link URLFixture} exposes a filesystem directory over HTTP. It is used in repository-url
  * integration tests to expose a directory created by a regular FS repository.
  */
 public class URLFixture extends AbstractHttpFixture {
```

Comment:
This is a fixture. I was wondering if we could plug in Nginx or some other simple http server to really make sure it works with at least a proper http server.

Reply:
That's a good idea, I'll add a real http server to the test suite 👍